transactionaldriver-jpa-and-tomcat/README.md

```diff
@@ -0,0 +1,27 @@
+### Description
```

Comment:
Is it worth mentioning somewhere that that the quicksart uses the connection pooling features provided by commons-dbcp